app/resources/app-base.html.template

```diff
@@ -13,8 +13,15 @@
 {% block title %}{{block.super}}{% endblock title %}
 
 {% block logo %}
-  <button class="menu" onclick="menuOpenNav()"><img src="{{env.global_url}}/menu.png" alt="Menu"></button>
-  <a href="{{env.global_url}}">Google Person Finder</a>
+  {% if env.repo and env.action and env.action != "start" %}
+    {% if env.rtl %}
+	  <button class="title-bar-button"><a href="{{env.start_url}}"><img src="{{env.global_url}}/forward_arrow.png" alt=""></a></button>Google Person Finder
```

Comment:
"Google Person Finder" part is common among all cases. Can you put it outside of these if-statements so that you write it just once?